src/main/java/org/jolokia/docker/maven/assembly/DockerFileDictionaryEnum.java

```diff
@@ -0,0 +1,13 @@
+package org.jolokia.docker.maven.assembly;
+
+/**
+ * @author  Paris Apostolopoulos <javapapo@mac.com></javapapo@mac.com> on 6/13/15.
```

Comment:
Git blame will remember the author, also you put the email as a xml tag (IntelliJ autocomplete I assume). :)